src/fsharp/vs/IncrementalBuild.fsi

```diff
@@ -19,7 +19,8 @@ type internal FSharpErrorSeverity =
     | Error
 
 [<Class>]
-type internal FSharpErrorInfo = 
+type internal FSharpErrorInfo =
+    member Id: string
```

Comment:
It is basically a string representation like the one C# has (example: CS0234). Roslyn APIs deal with strings in this form. I'll change it to an error code function and add a comment to make it clearer.